src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutAddressingContext.php

```diff
@@ -294,6 +294,14 @@ public function iSpecifyThePasswordAs($password)
         $this->addressPage->specifyPassword($password);
     }
 
+    /**
+     * @Then I should see :displayName in checkout header
+     */
+    public function iShouldSeeInCheckoutHeader(string $displayName): void
+    {
+        Assert::contains($this->selectShippingPage->getPurchaserEmail(), $displayName);
```

Comment:
As it does not have to be necessary **email**, maybe the method name should be changed? `getPurchaserIdentifier` (or sth similar)?